Jenkinsfile_k8s

```diff
@@ -1 +1,18 @@
-buildDockerAndPublishImage('jenkinsio', [mainBranch: 'master'])
+#!/usr/bin/env groovy
+
+/* Only keep the 10 most recent builds. */
+def projectProperties = [
+    [$class: 'BuildDiscarderProperty',strategy: [$class: 'LogRotator', numToKeepStr: '5']],
+]
+
+def principalBranch = 'master'
+
+/* Start by building the image, and eventually pushing it if the execution is on the master branch */
+buildDockerAndPublishImage('jenkinsio', [mainBranch: principalBranch])
+
+withCredentials([string(credentialsId: 'updatecli-github-token', variable: 'UPDATECLI_GITHUB_TOKEN')]) {
+  updatecli(action: 'diff')
```

Comment:
do we need updatecli to generate a different every since all we would ever be doing is updating the image tag right?

Reply:
oh that is a good point. As I initially added the configuration like we do on the `docker-*` repositories, it's a "copy and paste".